authentication/api_gateway/serializers.py

```diff
@@ -33,9 +34,14 @@ def create(self, validated_data):
         legal_address_data = validated_data.pop("legal_address")
         user_profile_data = validated_data.pop("user_profile", None)
         with transaction.atomic():
-            user.openedx_user.edx_username = username
             user.name = name
             user.save()
+            if user.openedx_user is None:
+                create_user(user)
```

Comment:
Noting this for me later - this calls the openedx create_user, but the user may not have an OpenEdxUser so they may not have a username yet.. not 100% sure here yet; have to fix my local Tutor setup, so this is a note for when I look at this next (prob Friday).